src/docs/content/docs/configuration/reaper_specific.md

```diff
@@ -0,0 +1,343 @@
+
+    autoScheduling:
+      enabled: true
+      initialDelayPeriod: PT15S
+      periodBetweenPolls: PT10M
+      timeBeforeFirstSchedule: PT5M
+      scheduleSpreadPeriod: PT6H
+      excludedKeyspaces: [myTTLKeyspace, ...]
+
+Definitions for the above sub-settings are as follows.
+
+#### `endabled`
+
+Type: *Boolean*
+
+Default: *false*
+
+Enables or disables `autoScheduling`.
+
+#### `initialDelayPeriod`
+
+Type: *String*
+
+Default: *PT15S* (15 seconds)
+
+The amount of delay time before the schedule period starts.
+
+#### `periodBetweenPolls`
+
+Type: *String*
+
+Default: *PT10M* (10 minutes)
+
+The interval time to wait before checking whether to start a repair task.
+
+#### `timeBeforeFirstSchedule`
+
+Type: *String*
+
+Default: *PT5M* (5 minutes)
+
+Grace period before the first repair in the schedule is started.
+
+#### `scheduleSpreadPeriod`
+
+Type: *String*
+
+Default: *PT6H* (6 hours)
+
+The time spacing between each of the repair schedules that is to be carried out.
+
+#### `excludedKeyspaces`
+
+Type: *Array* (comma separated *Strings*)
+
+The Keyspaces that are to be excluded from the repair schedule.
+
+</br>
+
+### `datacenterAvailability`
+
+Type: *String*
+
+Default: *ALL*
+
+Indicates to Reapers its deployment in relation to cluster data center network locality. The value must be either **ALL**, **LOCAL**, or **EACH**. Note that this setting controls the behavior for metrics collection.
+
+For security reasons, it is possible that Reaper will have access limited to nodes in a single datacenter via JMX (multi region clusters for example). In this case, it is possible to deploy an operate an instance of Reaper in each datacenter where each instance only has access via JMX (with or without authentication) to the nodes in its local datacenter. Where multiple instances of Reaper are in operation in this configuration, only the Apache Cassandra storage option can be used with Reaper. All other storage options are unsuitable in this case. This is because Reaper instances will rely on lightweight transactions to get leadership on segments before processing them. In addition, Reaper will check the number of pending compactions and actively running repairs on all replicas prior to processing a segment.
+
+**ALL** - requires Reaper to have access via JMX to all nodes across all datacenters. In this mode Reaper can be backed by all available storage types.
+
+**LOCAL** - requires Reaper to have access via JMX to all nodes only in the same datacenter local to Reaper. A single Reaper instance can operate in this mode and repair its local data center. In this case, can be backed by all available storage types and repairs to any remote datacenters are be handled internally by Cassandra. A Reaper instance can be deployed to each datacenter and be configured to operate in this mode. In this case, Reaper can only use Apache Cassandra as its storage. In addition, metrics can be collected asynchronously through the Apache Cassandra storage.
```

Comment:
> A single Reaper instance can operate in this mode and repair its local data center.

I'd rather say : and trigger repairs from within its local data center.
Otherwise the text suggests that it'll only repair the local DC.

Reply:
Good point. Changed.